src/N3Parser.js

```
@@ -524,20 +524,20 @@ export default class N3Parser {
   // ### `_completeLiteral` completes a literal with an optional datatype or language
   _completeLiteral(token) {
     // Create a simple string literal by default
-    let literal = this._literal(this._literalValue);
+    let literal = this._factory.literal(this._literalValue);
 
     switch (token.type) {
     // Create a datatyped literal
     case 'type':
     case 'typeIRI':
       const datatype = this._readEntity(token);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)